test/run-drun/ok/issue-3910.diff-low.ok

```diff
@@ -5,4 +5,4 @@
  f1
  f2
 -issue-3910.mo:17.10-17.35: execution error, argument value {trap = true} does not match parameter list
-+issue-3910.mo:17.20-17.32: execution error, value true does not match pattern
```

Comment:
I get that the change in the diagnostic is due to the fact that the whole pattern gets reported whereas before only the failing field did...